go/chat/server.go

```diff
@@ -3219,3 +3219,52 @@ func (h *Server) DismissJourneycard(ctx context.Context, arg chat1.DismissJourne
 		return fmt.Errorf("got %v conversations but expected 1", len(inbox.ConvsUnverified))
 	}
 }
+
+const welcomeMessageName = "__welcome_message"
+
+func (h *Server) welcomeStorage() types.ConversationBackedStorage {
```

Comment:
Is it ok to make the storage struct on-demand, or should I make a long lived WelcomeMessageManager on G containing this storage struct?